shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/external/CircusTrainConfig.java

```diff
@@ -23,35 +23,26 @@
 import java.util.List;
 import java.util.Map;
 
-import org.apache.hadoop.hive.metastore.api.FieldSchema;
-
 import com.google.common.base.Joiner;
 
-import com.hotels.bdp.circustrain.core.conf.ReplicaCatalog;
-import com.hotels.bdp.circustrain.core.conf.ReplicaTable;
-import com.hotels.bdp.circustrain.core.conf.ReplicationMode;
-import com.hotels.bdp.circustrain.core.conf.SourceCatalog;
-import com.hotels.bdp.circustrain.core.conf.SourceTable;
-import com.hotels.bdp.circustrain.core.conf.TableReplication;
+import com.hotels.bdp.circustrain.api.conf.ReplicaTable;
+import com.hotels.bdp.circustrain.api.conf.ReplicationMode;
+import com.hotels.bdp.circustrain.api.conf.SourceCatalog;
+import com.hotels.bdp.circustrain.api.conf.SourceTable;
+import com.hotels.bdp.circustrain.api.conf.TableReplication;
+import com.hotels.shunting.yard.replicator.exec.conf.ReplicaCatalog;
 
 public class CircusTrainConfig {
 
   private static final Joiner AND_JOINER = Joiner.on(" AND ");
   private static final Joiner OR_JOINER = Joiner.on(") OR (");
 
-  private static String partCondition(FieldSchema partitionColumn, String partitionValue) {
-    String type = partitionColumn.getType().toLowerCase();
-    boolean needQuotes = type.equals("string") || type.equals("char") || type.equals("varchar") || type.equals("date");
-    return new StringBuilder(partitionColumn.getName())
-        .append("=")
-        .append(needQuotes ? "'" : "")
-        .append(partitionValue)
-        .append(needQuotes ? "'" : "")
-        .toString();
+  private static String partCondition(String partitionColumn, String partitionValue) {
```

Comment:
NABD but `partitionCondition` would be a better name for this method as `part` can mean a lot of different things.

Reply:
ok, done